### ISSUE TITLE
Error about loss during training

### ISSUE THREAD

**Issue by FJ1546560200** [Zx55/dmcp#14]
Why is the loss value in max_width much greater than in min_width and random_width during the training process? For example, in max_width, the loss is 59, while the other two have a loss of only about 0.001

**Comment by Aqiu1123456** [comment]
请问一下你有没有添加预训练权重 pretrain_weight.pth 在训练过程中我的精度Pre特别小，是不是因为没有加预训练权重，但是不知道是不是官方的预训练权重是可用，我还没有尝试 就想来问一下你的效果如何
